frontend/src/stories/testQueue/QueueItem.stories.tsx

```diff
@@ -0,0 +1,72 @@
+import { Story, Meta } from "@storybook/react";
+
+import QueueItem, { QueueItemProps } from "../../app/testQueue/QueueItem";
+import { getMocks, StoryGraphQLProvider } from "../storyMocks";
+
+export default {
+  title: "App/Test Queue/Queue Item",
```

Comment:
Update to sentence case:
```suggestion
  title: "App/Test queue/Queue item",
```